camayoc/tests/qcs/cli/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""Test utilities for quipucords' ``qpc`` tests."""
+from io import BytesIO
+
+import pexpect
+import pytest
+
+from camayoc.config import get_config
+
+
+@pytest.fixture()
+def qpc_server_config():
+    """Read Camayoc's configuration file and configure qpc.
+
+    Both hostname and port must be available on the configuration file, for
+    example::
+
+        qpc:
+          hostname: localhost
```

Comment:
In the current configuration file format it is expected that hostname is actually a URL like `http://localhost:8000` this expects a different format where each information will have its own field, for example hostname, port and scheme (either http or https). The other option is to read a field called url and then use urlparse to extract the proper information. What is your opinion?

I am looking for a more meaningful approach and will be happy to update everything needed to do this refactor.

Reply:
I'm fine with breaking it up, but maybe we should just use a boolean for "https: true"

Reply:
That is an option, I don't have any preference here. If you think a boolean is better then that is fine with me